### ISSUE TITLE
Calling focus() on an element without an id (or tabindex) silently fails

### ISSUE THREAD

**Issue by reubenfirmin** [rjaros/kvision#241]
This is a browser (chrome) issue, but perhaps kvision can be smart.

To reproduce:

```

class MyContainer: Div() {

  init {
        style {
        //    id = "needed"
            tabindex = 1
        }

        onEvent {
            keydown = { event ->
                if (event.ctrlKey && event.keyCode == 222) { // ctrl '
                    console.log("hello world"
                }
            }
       }
       window.setTimeout( { focus() }, 40)
  }
}
```

Suggestion: check if element has id/tabindex in focus() function, and dynamically add if not. IMO they're an implementation detail that the user shouldn't have to care about.

**Comment by rjaros** [comment]
I've made a simple change for the `Tag` component, so it should work for all HTML elements. But the change doesn't affect more general components and containers (eg. `SimplePanel`), because in my opinion it could lead to incompatibility issues.

**Comment by rjaros** [comment]
Fixed in 4.2.2.
